pkg/image/registry/imagestreamtag/rest.go

```diff
@@ -321,6 +339,7 @@ func newISTag(tag string, imageStream *imageapi.ImageStream, image *imageapi.Ima
 	event := imageapi.LatestTaggedImage(imageStream, tag)
 	if event == nil || len(event.Image) == 0 {
 		if !allowEmptyEvent {
+			glog.V(4).Infof("did not find tag %s in image stream status tags: %#v", tag, imageStream.Status.Tags)
```

Comment:
wish we have some pretty-printer for tags (we print the tags in several places and the Status.Tags is pretty big arr)